src/kuduClient.ts

```diff
@@ -98,13 +100,25 @@ export class KuduClient {
         });
     }
 
+    getLogStream(): request.Request {
+        const baseUrl = `https://${this.webAppName}.${this.domain}/`;
+        const headers = {
+            Authorization: 'Basic ' + new Buffer(this.publishingUserName + ':' + this.publishingPassword).toString('base64')
+        };
+        const r = request.defaults({
+            baseUrl: baseUrl,
+            headers: headers,
+        });
+        return r('/api/logstream');
+    }
+
     private removeHomeFromPath(path: string): string {
```

Comment:
Can you remove this while you're here? I don't see it used...

Reply:
This was something I wrote for something reason.  I'll get rid of it in my next PR.